### PR TITLE
refactor: Kotlinify Keyboard

### PR DESCRIPTION
**Pull request by nopdan**
#### Issue tracker

Main changed files:
- Key.java
- Keyboard.java
- KeyboardView.java
- Event.java

#### Feature
Describe features of pull request

#### Code of conduct
- [x] [CONTRIBUTING](CONTRIBUTING.md)

#### Style lint
- [x] `make sytle-lint`

#### Build pass
- [x] `make debug`

#### Manually test
- [x] Done

#### Code Review
1. No wildcards import
2. Manual build and test pass
3. GitHub action ci pass
4. At least one contributor reviews and votes
5. Can be merged clean without conflicts
6. PR will be merged by rebase upstream base

#### Daily build
Login and download artifact at https://github.com/osfans/trime/actions

### CODE REVIEW

**Comment by nopdan** [comment]
提交只是将代码转为kotlin，几乎没有逻辑修改，目前没发现明显 bug。
鉴于这部分问题不少，最好还是重构一下。

**Comment by WhiredPlanck** [comment]
@nopdan 请尽量重新整理一下你的提交信息

File: app/src/main/java/com/osfans/trime/ime/keyboard/Event.java
Comment (WhiredPlanck) [inline app/src/main/java/com/osfans/trime/ime/keyboard/Event.java:1]: 是不是忘记改名了 :rofl: 这是 Java 的脸 Kotlin 的身

File: app/src/main/java/com/osfans/trime/ime/keyboard/Key.java
Comment (WhiredPlanck) [inline app/src/main/java/com/osfans/trime/ime/keyboard/Key.java:1]: 同上

File: app/src/main/java/com/osfans/trime/ime/keyboard/Keyboard.java
Comment (WhiredPlanck) [inline app/src/main/java/com/osfans/trime/ime/keyboard/Keyboard.java:1]: 同上

File: app/src/main/java/com/osfans/trime/ime/keyboard/KeyboardView.java
Comment (WhiredPlanck) [inline app/src/main/java/com/osfans/trime/ime/keyboard/KeyboardView.java:1]: 同上

**Review by WhiredPlanck — CHANGES_REQUESTED** [review]

**Review by WhiredPlanck — APPROVED** [review]

**Pull request merged**